src/Controller/EnMarche/CitizenProjectController.php

```diff
@@ -260,4 +262,40 @@ public function unfollowAction(Request $request, CitizenProject $citizenProject)
             ],
         ]);
     }
+
+    /**
+     * @Route("", name="app_citizen_project_landing")
+     * @Method("GET")
+     */
+    public function landingAction(): Response
+    {
+        $city = $this->getUser() ? $this->getUser()->getCityName() : SearchParametersFilter::DEFAULT_CITY;
+
+        return $this->render('citizen_project/landing.html.twig', [
+            'city' => $city,
+        ]);
+    }
+
+    /**
+     * @Route("/landing/results", name="app_citizen_project_landing_result", condition="request.isXmlHttpRequest() and request.query.get('city')")
+     * @Method("GET")
+     */
+    public function landingPageResultAction(Request $request, CitizenProjectManager $citizenProjectManager): Response
+    {
+        $search = $this->get(SearchParametersFilter::class);
```

Comment:
Pourquoi `SearchParametersFilter` est un service ? Ça n'a pas de sens car c'est un objet qui transporte de la donnée.

Reply:
Oui on est d'accord on a eu la même discussion avec @HeahDude :D

Reply:
Ça a été fait comme ça y'a un moment et on a pas voulu tout changer

Reply:
Ignore this comment for now, as explained by @B-Galati.